app/models/movie.rb

```diff
@@ -0,0 +1,15 @@
+class Movie < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  validates :inventory, presence: true, numericality: { only_integer: true }
+  before_create :set_available
+  before_save :set_available
```

Comment:
I think this might cause issues. 

When we use `before_save` that means the `set_available` callback will be run before _every_ time we save (including when the movie is updated). As such, we would never have an available inventory which was not equal to the inventory.

For this case I think just sticking with `before_create` is sufficient.